### DISCUSSION TITLE
Moving MedicaCMS to a EC2

### DISCUSSION THREAD

**Discussion by Evelyn-Mend** [mediacms-io/mediacms#707] [Q&A]
I am doing a project to move MediaCMS from my local matching to the cloud (AWS-EC2). I am trying to move it to EC2 but having some trouble with how to start it. I created an instance in EC2 after that I don't know what to do next. Can you give me some help or know another service I can use?

**Comment by mgogoulos** [comment]
Hi, without providing any info about what is going wrong it is hard to help!

What have you tried, and what are the errors?
